src/components/RelationsOverlay/BoundingBox.js

```diff
@@ -101,8 +102,13 @@ const _detect = region => {
       };
     }
     case "brushregion": {
-      const points = flatten(wrapArray(region.touches.filter(t => t.type === "add").map(t => Array.from(t.points))));
-      return imageRelatedBBox(region, Geometry.getBrushBBox(points));
+      return imageRelatedBBox(
+        region,
+        Geometry.scaleBBox(
+          Geometry.getImageDataBBox(region.imageData.data, region.imageData.width, region.imageData.height),
```

Comment:
Image might be scaled down by the browser (we have default restrictions on that that can be overwritten by the config). It seems that `naturalWidth` and `naturalHeight` should be used instead as these values refer to the real image dimensions of the image.

Not sure how it will affect the calculation though, so it worth testing, I might be wrong here.